stubs/laravel-backed-enum.stub

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace {{ namespace }};
+
+use Webfox\LaravelBackedEnums\BackedEnum;
+use Webfox\LaravelBackedEnums\IsBackedEnum;
+
+enum {{ class }}: {{ type }} implements BackedEnum
+{
+    use IsBackedEnum;
+
+    /**
+     * Add your Enums below using.
+     * e.g. case Standard = 'standard';
```

Comment:
What if we do `e.g. {{ example }}`
And then replace example with `case Standard = 'standard';` or `case Standard = 0;` same as we do the type?